modules/meeting/app/controllers/meetings_controller.rb

```diff
@@ -123,14 +128,24 @@ def set_time_zone(&)
 
   def build_meeting
     @meeting = Meeting.new
+    if meeting_params.present?
+      convert_params
+      @meeting.participants.clear # Start with a clean set of participants
+      @meeting.participants_attributes = @converted_params.delete(:participants_attributes)
+      @meeting.attributes = @converted_params
+    end
     @meeting.project = @project
     @meeting.author = User.current
   end
 
+  def project_id
+    params[:project_id] || params.dig(:meeting, :project_id)
+  end
+
   def find_optional_project
-    return true unless params[:project_id]
+    return true if project_id.blank?
 
-    @project = Project.find(params[:project_id])
+    @project = Project.find(project_id)
     authorize
   rescue ActiveRecord::RecordNotFound
```

Comment:
Please check whether this method overwrite over what is already implemented in [the `ApplicationController`](https://github.com/opf/openproject/blob/dev/app/controllers/application_controller.rb#L284-L288) is actually necessary.

Reply:
It is no longer necessary by using the `op-project-autocompleter` as I can have `project_id` be a non-nested param within the `meeting` key.